lib/layer/data.py

```diff
@@ -1322,6 +1302,22 @@ def _load_strokemap_from_ora(self, elem, x, y, orazip=None, oradir=None):
         else:
             raise ValueError("either orazip or oradir must be specified")
 
+    def get_paintable(self):
+        """True if this layer currently accepts painting brushstrokes"""
+        return (
+            self.visible and
+            not self.locked and
+            self.branch_visible and
+            not self.branch_locked
+        )
+
+    def get_fillable(self):
+        """True if this layer currently accepts flood fill"""
```

Comment:
Would it be possible to add some doctests for these?

Not sure if it's an error or not, but why allow filling on functionally invisible layers?